mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go.tmpl

```diff
@@ -1051,13 +1052,13 @@ func ResourceBigQueryTable() *schema.Resource {
 						},
 
 						// UseLegacySQL: [Optional] Specifies whether to use BigQuery's
-						// legacy SQL for this view. The default value is true. If set to
-						// false, the view will use BigQuery's standard SQL:
+						// legacy SQL for this view. If set to false, the view will use
+						// BigQuery's standard SQL:
 						"use_legacy_sql": {
 							Type:        schema.TypeBool,
 							Optional:    true,
-							Default:     true,
-							Description: `Specifies whether to use BigQuery's legacy SQL for this view. The default value is true. If set to false, the view will use BigQuery's standard SQL`,
+							Computed:    true,
```

Comment:
I see we are using Computed: true
Is there a reason for this? This basically mean that the field is managed upstream by the API when unset.
Specifically it is tricky because there will be no way to unset this field is we go forward in this manner.
If this is expected/anticipated we are good to go.
I just want to double confirm because if you need tri state behavior (unset/false/true) this will be difficult to maintain when the user cannot unset the value

Reply:
discussed offline, api behaves similarly